hot-desk/src/main/java/com/exadel/sandbox/security/model/ApiResponse.java

```diff
@@ -0,0 +1,27 @@
+package com.exadel.sandbox.security.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+
+@NoArgsConstructor
+public class ApiResponse {
+    private String message;
+    private boolean success;
+    private Object token;
```

Comment:
Why it Object?

Reply:
To be able to use when the return status of various objects is observed in the future